lib/chain.js

```diff
@@ -161,7 +161,9 @@ function call(handler, err, req, res, _next) {
         return;
     } else if (!hasError && arity < 4) {
         // request-handling middleware
-        handler(req, res, next);
+        process.nextTick(function nextTick() {
```

Comment:
Is this enough to reset the FG stack? Or do we need setImmediate?

Reply:
cc @misterdjules

Reply:
What do we mean by "FG stack"? If the question is whether the `handler` call's stack frame would not have the previous handler as a parent stack frame, the answer is yes, it would not have the previous handler as a parent stack frame.

Reply:
Ah Peter clarified that FG is for Flame Graphs :)